web/styles/popovers.css

```diff
@@ -568,26 +568,22 @@ ul {
 
     .user_profile_manage_own_edit_button,
     .user_profile_manage_others_edit_button {
-        width: 25px;
-        height: 14px;
+        width: 22px;
+        height: 18px;
```

Comment:
What is the reason for these `width` and `height` changes? I tried resetting those to their original values and removed the `margin-top` properties for both the icons and here is the screenshot, which I guess looks fine -
![Screenshot from 2024-03-28 12-20-47](https://github.com/zulip/zulip/assets/35494118/b07bcfba-28a5-47ea-b7cf-2dbd73caf7dd)

Reply:
We need to set the height and width at least equal to the corresponding font size otherwise the child container(in this case `<i>` tag) will come out of the parent container(in this case `<a>` tag) as you can see in the below images :

Original Height and Width settings (without Margin-top) :
![Screenshot from 2024-04-01 02-05-02](https://github.com/zulip/zulip/assets/128511266/854eaf9a-7b53-4cdb-875a-e770381c4740)

Changed Height and Width settings (With Margin top) :
![Screenshot from 2024-04-01 02-06-18](https://github.com/zulip/zulip/assets/128511266/df4524b8-2a0c-434c-8325-8847c53716e1)

Reply:
Ok.
Having this custom values for height and width does not seem good and would be better to remove them. I see that removing height breaks the alignment vertically so maybe we could use something like `vertical-align` on the icon. And changing width changes the space between the icons, but would be good to use `margin` to adjust the space between icons.

@karlstolley thoughts on refactoring this?

Reply:
I've tried using `vertical-align` and `margin` and it is providing the required results. Should I proceed with refactoring in a separate commit ? @sahil839 @karlstolley

Reply:
What would be really nice to see here is fuller use of flexbox--no horizontal margins, but gaps instead. That could include moving the status indicator into the flexbox as well. Vertical centering is going to be tough to pull off because there are even-height icons sitting on an odd height (27.5px) line-height. If usernames don't wrap in this modal ever, it might be better to just set `line-height: 28px` and be done with it.

Reply:
Yes, we do not wrap names here and just show ellipsis for very long names.

Reply:
@sahil839, Please let me know if this works for you.

Reply:
The link icon looks like it's above the other one.

Reply:
I've updated the font size to fix vertical alignment. Updated the screenshot in the PR as well. 

Also, Both can't have the same font size due to their shapes and predefined padding, and if we are aligning them vertically centered then one icon is bound to be above other.